### ISSUE TITLE
Cannot read property filter of undefined

### ISSUE THREAD

**Issue by zepod** [zalmoxisus/mobx-remotedev#19]
I encountered following error, using mobx-remotedev.

![mobx-err](https://cloud.githubusercontent.com/assets/19804724/23305278/4bafc1b2-fa9e-11e6-8f0b-8aea5f81b402.png)

after removing mobx-remotedev from my module, no error was triggered. Here is my store (module)

```js
// @flow
import remotedev from 'mobx-remotedev'
import {observable} from 'mobx'

class UI {
  @observable preloading :boolean

  constructor() {
    this.preloading = false
  }

  startPreloading() {
    this.preloading = true // <- line 30, thus error comes from here
  }
}

export default remotedev(UI)
```

I also created StackOverflow question, regarding this topic, so if it isn't mistake on my part, it'd be appreciated, if you'd elaborate on how to resolve this gracefully.

**Comment by zalmoxisus** [comment]
Sorry, I broke it in the yesterday release of Redux DevTools Extension. Forgot that we're using it here (via API) as well. Fixed in `2.14.1`, which will be available on Chrome Store in maximum an hour.

**Comment by whu-iss-jack** [comment]
I also encountered this problem, and the chrome debugger navigate me to files with name like VM..
It's the problem of the redux extension,window.devToolsOptions  not  initialized yet?

**Comment by zalmoxisus** [comment]
@whu-iss-jack, yes. Please make sure that you have Redux DevTools Extension `2.14.1` on `chrome://extensions` page.
